### PR TITLE
SPR-5509 prepare for dockerize

### PR DESCRIPTION
**Pull request by MozhouGaoS**
[JIRA](https://sensorup.atlassian.net/browse/SPR-5509)

## Summary

1. Updated to latest LDAR-Sim  
2. Added pipfile, pipfile lock, and some test programs file

### CODE REVIEW

**Comment by MozhouGaoS** [comment]
@jeremysquires-sensorup I Just finished building the deployment package. Feel free to review it. Thanks.

File: LDAR_Sim/src/utils/generic_functions.py [hunk 1]
```diff
@@ -207,46 +207,46 @@ def ecef_to_llh(ecef_km):
     return lat, lon, h
 
 
-def init_orbit_poly(predictor, T1, T2, interval):
```
Comment (jeremysquires-sensorup) [inline LDAR_Sim/src/utils/generic_functions.py]: Why are you removing the satellite functionality? If it is not called, it won't be used, right?

**Comment by jeremysquires-sensorup** [comment]
@MozhouGaoS - you're removed a lot of the configurations, and the satellite implementation code. Was this to make the run time and size shorter? I don't think I would remove them, I think I would move them into a non-active configuration, like in the archive where they won't be loaded. We may very well want to have them as options in the future. We can put them under a test folder, for example.

**Comment by MozhouGaoS** [comment]
> @MozhouGaoS - you're removed a lot of the configurations, and the satellite implementation code. Was this to make the run time and size shorter? I don't think I would remove them, I think I would move them into a non-active configuration, like in the archive where they won't be loaded. We may very well want to have them as options in the future. We can put them under a test folder, for example.



> @MozhouGaoS - you're removed a lot of the configurations, and the satellite implementation code. Was this to make the run time and size shorter? I don't think I would remove them, I think I would move them into a non-active configuration, like in the archive where they won't be loaded. We may very well want to have them as options in the future. We can put them under a test folder, for example.

**Pull request closed without merging**